src/modules/roc_pipeline/receiver.cpp

```diff
@@ -146,11 +146,18 @@ bool Receiver::create_session_(const packet::PacketPtr& packet) {
         roc_log(LogError, "receiver: can't create session, unexpected non-udp packet");
         return false;
     }
+
+    if (packet->udp()->dst_addr == config_.repair_port.address) {
```

Comment:
This is not a correct way to check if the packet is repair (see comment above). The correct way would be the following:

```
if (packet->flags() & packet::Packet::FlagRepair) {
    ...
}
```

Reply:
455a825301accba5d3f79c7d45ef4315171d0a3f